app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,90 @@
+<div class="row inset">
+  <section class="col col-720">
+<p>Teaching computing you'll help pupils become responsible and confident users of information and technology.</p>
+<p>In an increasingly digital world, you could inspire pupils to look at a future role in technology. With roles in technology making up 14% of all job opportunities in the UK.</p>
```

Comment:
@gemmadallmandfeif if we include a stat do we need to reference the source somewhere? And would we need to say in 2023 or 2024 the date when the figure's from? as it may well then change.

Reply:
@caitlin-reidy had a chat with Gemma and we should take the stat out. Thanks